ifrs17/Test/ImportStorageTest.ipynb

```diff
@@ -1428,13 +1452,175 @@
+                "\n    }",
+                "\n    ",
+                "\n    await Workspace.DeleteAsync<IfrsVariable>(await Workspace.Query<IfrsVariable>().ToArrayAsync());",
+                "\n    await Workspace.DeleteAsync<RawVariable>(await Workspace.Query<RawVariable>().ToArrayAsync());",
+                "\n",
+                "\n    await Workspace.DeleteAsync<GroupOfInsuranceContract>(await Workspace.Query<GroupOfInsuranceContract>().ToArrayAsync());",
+                "\n    await Workspace.DeleteAsync<DataNodeState>(await Workspace.Query<DataNodeState>().ToArrayAsync());",
+                "\n    if(errors.Any()) ApplicationMessage.Log(Error.Generic, string.Join(\"\\n\", errors));",
+                "\n    return Activity.Finish();",
+                "\n}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var inputDataVariable = new RawVariable[]{ sampleRawVar with {DataNode = \"Gross1\"},",
+                "\n                                           sampleRawVar with {DataNode = \"Gross2\"},",
+                "\n                                         };",
+                "\nvar primaryScopeBm = new string[]{\"Gross1\", \"Gross2\", \"Paa1\", \"Paa2\", \"Paa3\"};",
+                "\nvar activity = await CheckPrimaryScopeAsync<RawVariable>(inputDataVariable, primaryScopeBm);",
+                "\nactivity"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "activity.Status.Should().Be(ActivityLogStatus.Succeeded);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var inputDataVariable = new RawVariable[]{ sampleRawVar with {DataNode = \"Paa1\"},",
+                "\n                                         };",
+                "\nvar primaryScopeBm = new string[]{\"Paa1\", \"Paa2\", \"Paa3\"};",
+                "\nvar activity = await CheckPrimaryScopeAsync<RawVariable>(inputDataVariable, primaryScopeBm);",
+                "\nactivity"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "activity.Status.Should().Be(ActivityLogStatus.Succeeded);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var inputDataSourceVariable = new IfrsVariable[]{ new IfrsVariable {DataNode = \"Paa1\", AocType = \"BOP\", Novelty = \"N\", Partition = sampleRawVar.Partition},",
```

Comment:
Minor comment: use of enums for AocType and Novelty.

Reply:
Same as above.